docs/pascal_voc_user_manual.md

```diff
@@ -0,0 +1,442 @@
+# person:255,0,0:head:
+```
+
+Also you can use original voc format of label map, for example:
+
+``` bash
+datum export -f voc_layout -- --label-map voc
+```
+
+## Export to Pascal VOC
+
+There are few ways to convert dataset to Pascal VOC format:
+
+``` bash
+# export dataset into Pascal VOC format (classification) from existing project
+datum export -p <path/to/project> -f voc -o <path/to/export/dir> -- --tasks classification
+
+# converting to Pascal VOC format from other format
+datum convert -if imagenet -i <path/to/imagenet/dataset> \
+    -f voc -o <path/to/export/dir> \
+    -- --label_map voc --save-images
+```
+
+Argument `--tasks` allow to specify tasks for export dataset,
+by default datumaro uses all tasks.
+Argument   `--label_map` allow to define user label map, for example
+
+``` bash
+# mycolormap.txt [label : color_rgb : parts : actions]:
+# cat:0,0,255::
+# person:255,0,0:head:
+datum export -f voc_segmentation -- --label-map mycolormap.txt
+```
+
+Or you can use original Pascal VOC label map:
+
+``` bash
+datum export -f voc_layout -- --label-map voc
+```
+
+## Datumaro functionality
+
+Datumaro supports filtering, transformation, merging and etc. for all formats
+and for the Pascal VOC format in particular. Follow
+[user manual](../docs/user_manual.md)
+to get more information about these operations.
+
+There are few examples of using datumaro operations to solve
+particular problems:
+
+- We can convert Pascal VOC dataset to Market-1501 format. But Market-1501 dataset
+only has a `person` class, marked with a bounding box. And to perform the conversion
+we could filter the Pascal VOC dataset. With datumaro we can do it like this
+
+``` bash
+# create datumaro project with Pascal VOC dataset
+datum import -o <path/to/voc/dataset> --name project
+
+# convert labeled shapes into bboxes
+datum transform -t shapes_into_boxes
+cd project-shapes_to_boxes
+
+# keep only person class items
+datum filter --mode items+annotations \
+    -e '/item[label="person"]' \
+    -o <path/to/output/project>
+
+# delete other labels from dataset
+datum trasnform -t remap_labels -- -l person:person --default delete \
+    -o <path/to/final/project>
+```
+
+To make sure that the converting was succesful we can check the output project:
+
+```bash
+cd <path/to/final/project>
+datum info
+```
+
+- Pascal VOC 2007 use about 900MB disk space, you can store half as much if keep
+only store the test subset, and with datumaro split the test subset into test
+and training subsets:
+
+```bash
+# create datumaro project with Pascal VOC 2007 (only test subset) dataset
+datum import -o ./VOC2007 --name project
+
+# split the test subset into test and training subsets
+datum transform -t random_split
+
+# or you can specify ratio for subsets (by default test:.67 train:.33)
+datum transform -t random_split -- -s train:.5 -s test:.5
+```
+
+- If you don`t need a variety of classes in Pascal VOC dataset,
+with datumaro you can group the classes for your task:
```

Comment:
```suggestion
with Datumaro you can group the classes for your task:
```

The following command won't group classes, it will rename and join them. I suggest to pick a better term for this.